app/AccountancyModule/PaymentModule/presenters/PaymentPresenter.php

```diff
@@ -118,7 +119,12 @@ public function actionDefault(int $id): void
             $this['pairButton']->setGroups([$id]);
         }
 
-        $nextVS = $this->model->getNextVS($group->getId());
+        try {
+            $nextVS = $this->model->getNextVS($group->getId());
+        } catch (InvalidVariableSymbol $exception) {
+            $this->flashMessage('Nelze vygenerovat následující VS: \'' . $exception->getMessage() . '\'', 'danger');
```

Comment:
Tohle pls radši do property:
```suggestion
            $this->flashMessage('Nelze vygenerovat následující VS: \'' . $exception->getInvalidValue() . '\'', 'danger');
```

```php
final class InvalidVariableSymbol extends InvalidArgumentException
{
    private string $invalidValue;
    
    public function __construct(string $invalidValue)
    {
        parent::__construct(sprintf('"%s" is not a valid variable symbol', $invalidValue));
        $this->invalidValue = $invalidValue;
    }
    
    public function getInvalidValue(): string
    {
        return $this->invalidValue;
    }
}
```

Takhle to vypadá, že see tam vypíše nějaká interní chybová hláška :slightly_smiling_face: